### PR TITLE
handle click on element to set focus

### PR DESCRIPTION
**Pull request by albertjan**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "processing.sk",
   "description": "A python interface for processing.js for skulpt",
   "main": "main.js",
-  "version": "0.0.10",
+  "version": "0.0.111",
```
Comment (eah13) [inline bower.json]: typo?

File: bower.json [hunk 2]
```diff
@@ -18,6 +18,6 @@
   ],
   "dependencies": {
     "skulpt": "^0.11.0",
-    "Processing.js": "git@github.com:trinketapp/processing-js.git#1.6.11"
+    "Processing.js": "git@github.com:trinketapp/processing-js.git#75ab032d4d1baf3acd14ed35e941be6f93a1904c"
```
Comment (eah13) [inline bower.json]: This is ready for a release in processing-js I believe.

File: bower.json (hunk 1 above)
Comment (albertjan) [reply bower.json]: definitely 👍

File: bower.json (hunk 2 above)
Comment (albertjan) [reply bower.json]: Just did that

**Pull request merged**